src/test/java/net/snowflake/client/jdbc/ChunkDownloaderS3RetryUrlLatestIT.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+ */
+package net.snowflake.client.jdbc;
+
+import static net.snowflake.client.AbstractDriverIT.getConnection;
+import static org.junit.Assert.*;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import net.snowflake.client.category.TestCategoryOthers;
+import net.snowflake.client.core.*;
+import org.apache.http.client.methods.*;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(TestCategoryOthers.class)
+public class ChunkDownloaderS3RetryUrlLatestIT {
+
+  private Connection connection;
+  private SFStatement sfStatement;
+  private SFBaseSession sfBaseSession;
+  private ChunkDownloadContext sfContext;
+
+  @Before
+  public void setup() throws SQLException, InterruptedException {
+    Properties paramProperties = new Properties();
+    // connection = getConnection("s3testaccount", paramProperties);
```

Comment:
Should this be changed back to the commented-out line so that s3testaccount is used? s3 is the default type of deployment for Jenkins at the moment but might be better to ensure it's always s3